solidstart-tanstackquery-tailwind-modules/src/components/UserProfile/OrgList.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from '@solidjs/router';
+import { For } from 'solid-js';
+
+interface OrgListProps {
+  organizations: {
+    avatarUrl: string;
+    login: string;
+  }[];
+}
+
+function OrgList(props: OrgListProps) {
+  const navigate = useNavigate();
+  return (
+    <div class="mt-5 border-t border-gray-200">
+      <h2 class="my-2 pt-2 text-gray-800 font-bold">Organizations</h2>
+      <div data-testid="profile page orgs" class="flex flex-wrap space-x-2">
+        <For each={props.organizations}>
```

Comment:
For the prop, can you check out solid js props
You can read here. 
https://www.solidjs.com/tutorial/props_defaults

Nothing wrong with your approach, but we wanna be more solid.
we have mergeProps and splitProps

Reply:
For this, the "props. organizations" is just direct - the only property the props has, no need to make use of split or merge props here

Reply:
Remember it's a learning ground for others who will be using it. That's why I said we should be more solidjs than react. your approach is not wrong at all.

Reply:
Yes, I get that, but there is no benefit in using the suggestion, unfortunately.